app/serializers/reviewable_serializer.rb

```diff
@@ -72,19 +72,19 @@ def self.payload_attributes(*attributes)
   end
 
   def attributes
-    data = super
+    super.tap do |data|
+      data[:removed_topic_id] = object.topic_id unless object.topic
```

Comment:
We should have a test that confirms that the serializer includes this attribute.